### ISSUE TITLE
python3-ev3dev v2 speed vs speed_pct

### ISSUE THREAD

**Issue by ndward** [ev3dev/ev3dev-lang-python#508]
ev3dev version: 4.14.61-ev3dev-2.2.2-ev3
ev3dev-lang-python version:
ii python3-ev3dev 1.2.0 all Python language bindings for ev3dev
ii python3-ev3dev2 2.0.0~beta1 all Python language bindings for ev3dev

Documentation for the new v2 motor commands is at https://python-ev3dev.readthedocs.io/en/ev3dev-stretch/motors.html

As I understand it, the new 'on' commands for **single motors** use '**speed_pct**' for the percentage of the rated maximum speed, and the new 'on' commands for **motor pairs** use '**speed**' for the same thing (referring to the faster of the two motors). It's extremely confusing to have two different parameter names for what is essentially the same thing. Its difficult to remember which name to use and difficult to understand why two different names are necessary.  I believe a number of v2 demo scripts don't actually work because of confusion between the two.

It's true that it's not necessary to include the names in commands since these arguments can be used as positional arguments e.g. we can write **drive.on_for_degrees(50, 60, 70)** but it's then not very clear what the arguments represent – we can't even know from this command whether they represent **(steering, speed, degrees)** or **(left _speed, right_speed, degrees)**  whereas the command **drive.on_for_degrees(steering=50,speed= 60, degrees=70)** removes the ambiguity and makes everything clearer. Therefore it is to be hoped that names will be included which means the problem of **speed** vs **speed_pct** is all the more significant.

Furthermore, @wasabifan has suggested to me that normally **speed** and **speed_pct** will be used in conjunction with the conversion functions such as **SpeedDPS(degrees_per_second)** which converts a value in degrees per second into the corresponding **speed** or **speed_pct** value. If you don’t use a  conversion function then commands such as **on(speed_pct=50)** will give different speeds (in degrees per second) for the medium motor and for the large motors since their rated maximum speeds are different.  But in a command such as **on(speed_pct(SpeedDPS(90))** where we giving a value in degrees per second isn't it confusing to see 'pct' in the code?

I therefore strongly suggest that the same parameter name 'speed' should be used for both the 'on' commands for **single motors** and the 'on' commands for **motor pairs**. (For the **MoveTank** commands that is **left_speed** and **right_speed**).

PS Just to confuse things still further, there is also a **speed** property which return the current motor speed in tacho counts per second and there is still also the **speed_sp** parameter which is for use with the soon-to-be-deprecated **run_*** functions.

**Comment by WasabiFan** [comment]
As mentioned via email, we have removed all `speed_pct` names. Anything that had that name in the past will now be called `speed`.

https://github.com/ev3dev/ev3dev-lang-python-demo/pull/40 Will handle the demo program updates.

**Comment by WasabiFan** [comment]
Fix released in `2.0.0~beta2`. To upgrade:

```
sudo apt-get update
sudo apt-get install --only-upgrade python3-ev3dev2
```
